src/searchbar.cpp

```diff
@@ -297,7 +297,9 @@ QModelIndex SearchBarLineEdit::getDefaulSuggestionIndex() const
     const auto firstSuggestionText = firstSuggestionIndex.data().toString();
```

Comment:
I appreciate the added `const` however while resolving merge conflicts during rebase you lost the the update to the above comment.

Reply:
See https://github.com/kiwix/kiwix-desktop/compare/07aa791ed7844595451c25b5bc313b9fd80b1923..80acbcaeb601eeb4813dbe6bd4e1fe66352b39b9

Reply:
Done